assignment3/assignment3_1.js

```diff
@@ -0,0 +1,19 @@
+//assignment1_1.js
+var exports=module.exports={};
+ exports.secondLargest=function(array) {
+ var first, second;
+    first = second = -99999;
+  var n=array.length;
+    for (var i = 0; i < n; i ++)
```

Comment:
What is `i`. Use something descriptive name, which describe what is this variable doing,